test/onnx/expect/TestOperators.test_stft_default.expect

```diff
@@ -0,0 +1,115 @@
+ir_version: 8
```

Comment:
I would remove all expect files as they are hard to maintain. Only

Reply:
Done!